ibm/mas_devops/roles/suite_app_configure/vars/iot.yml

```diff
@@ -2,6 +2,6 @@
 mas_app_ws_fqn: iotworkspaces.iot.ibm.com
 mas_app_ws_apiversion: iot.ibm.com/v1
 mas_app_ws_kind: IoTWorkspace
-
 mas_app_cfg_delay: 120
 mas_app_cfg_retries: 30
+mas_app_ws_spec: {}
```

Comment:
Bad merge?  Default specs are defined in a seperate file ... see vars/defaultspecs/xxx.yaml.